maintainers/maintainer-list.nix

```diff
@@ -14671,4 +14671,10 @@
     github = "snpschaaf";
     githubId = 105843013;
   };
+  jali-clarke = {
+    email = "jinnah.ali-clarke@outlook.com";
+    name = "Jinnah Ali-Clarke";
+    github = "jali-clarke";
+    githubId = 17733984;
+  };
```

Comment:
Sorry for making this a bit more complicated but according to guidelines adding of new maintainer should be done in separate commit. This means that this PR should contain 2 commits.

- add jali-clarke to maintainers list
- elmPackages.elm-pages: init at 2.1.11